src/App.tsx

```diff
@@ -1,24 +1,130 @@
 /* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useState } from 'react';
 
-const USER_ID = 0;
+import { USER_ID } from './helpers/userId';
+import { Todo } from './types/Todo';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { UserWarning } from './UserWarning';
+import { Footer } from './components/Footer';
+import * as todoService from './api/todos';
+import { ErrorMessages } from './components/ErrorMessages';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filteredTodos, setFilteredTodos] = useState<Todo[]>([]);
```

Comment:
I think there is no need to create a separate state for filtered todos. You can just use useMemo instead